splunk_eventgen/lib/eventgensamples.py

```diff
@@ -302,61 +301,52 @@ def latestTime(self):
     def utcnow(self):
         return self.now(utcnow=True)
 
-    def _openSampleFile(self):
-        logger.debug("Opening sample '%s' in app '%s'" % (self.name, self.app))
-        self._sampleFH = open(self.filePath, 'rU')
-
-    def _closeSampleFile(self):
-        logger.debug("Closing sample '%s' in app '%s'" % (self.name, self.app))
-        self._sampleFH.close()
-
```

Comment:
Remove the `self._sampleFH` which is a file handler and can not be pickled. This will cause `PickleError` in Python3.